insights/parsers/qemu_xml.py

```diff
@@ -140,19 +178,130 @@ class QemuXML(XMLParser):
         >>> memnode[1].get('mode') == 'strict'
         True
     """
-    def parse_dom(self):
-        if self.dom is None:
-            return
-        else:
-            domain = {}
-            for child in self.dom:
-                if len(child) == 0:
-                    domain[child.tag] = child.text
-                else:
-                    domain[child.tag] = [c.items() for c in child]
+    pass
 
-            return domain
 
-    @property
-    def vm_name(self):
-        return self.data.get('name', None)
+@parser(Specs.var_qemu_xml)
+class VarQemuXML(BaseQemuXML):
+    """This class parses xml files under ``/var/run/libvirt/qemu/`` using
+    ``QemuXML`` base parser.
+
+    Sample file::
```

Comment:
Please simplify the content of the `Sample file::`, since these `pydoc` will be printed on the user's console, please make it as shorter as it is.